docs/docs/getting-started/cli.md

```diff
@@ -0,0 +1,247 @@
+
+:::info
+If you have an existing trace data source, [read this](../configuration/overview.md), or see how to use OpenTelemetry Collector [here](../configuration/connecting-to-data-stores/opentelemetry-collector).
+:::
+
+### 5. Configure OpenTelemetry Collector
+
+Next, you'll be prompted about OpenTelemetry Collector. Proceed with `N`.
+
+```text title="Expected output:"
+Do you have an OpenTelemetry Collector? [y/N]:
+```
+
+Now we prompt you if you want us to generate an OpenTelemetry Collector config for you. Select `Y`.
+
+```text title="Expected output:"
+Do you want me to set up one? [Y/n]:
+```
+
+This will generate an OpenTelemetry Collector config for you.
+
+:::info
+Want to read in more detail how to use OpenTelemetry Collector? [Check this out](../configuration/connecting-to-data-stores/opentelemetry-collector).
+:::
+
+### (Optional) Enable a demo app
+
+In the next step, the CLI will ask if you want a demo app to try out your Tracetest config.
+
+```text title="Expected output:"
+Do you want to enable the PokeShop demo app? (https://github.com/kubeshop/pokeshop/) [Y/n]:
+```
+
+Proceed with `Y` again.
+
+### 6. Select output directory for Tracetest server installation
+
+Lastly, select the output directory where to store all files related to Tracetest. 
+
+```text title="Expected output:"
+Tracetest output directory [tracetest/]:
+```
+
+Proceeding will set it to `tracetest/` by default.
+
+```text title="Expected output:"
+-> Thanks! We are ready to install TraceTest now
+
+ SUCCESS  Install successful!
+
+To start tracetest:
+
+	docker compose -f docker-compose.yaml -f tracetest/docker-compose.yaml up -d
+
+Then, use your browser to navigate to:
+
+  http://localhost:11633
+
+Happy TraceTesting =)
+```
+
+:::note
+This means all Tracetest-related config files, files for Docker Compose, and any config file for OpenTelemetry Collector will be located in a `./tracetest/` directory, with a `docker-compose.yaml` file located in the root directory where you ran the `tracetest server install` command.
+:::
+
+### 7. Start Docker Compose
+
+```bash
+docker compose -f docker-compose.yaml -f tracetest/docker-compose.yaml up -d
+```
+
+```bash title="Condensed expected output from the Tracetest container:"
+Starting tracetest ...
+...
+2022/11/28 18:24:09 HTTP Server started
+...
+```
+
+### 8. Open the Tracetest Web UI
+
+Open your browser on [`http://localhost:11633`](http://localhost:11633).
+
+Create a [test](../web-ui/creating-tests.md).
+
+:::info
+Running a test againts `localhost` will resolve as the 127.0.0.1 inside the Tracetest container. To run tests againts apps running on your local machine, add them to the same network and use service name mapping instead. Example: Instead of running an app on `localhost:8080`, add it to your Docker Compose file, connect it to the same network as your Tracetest service, and use `service-name:8080` in the URL field when creating an app.
```

Comment:
```suggestion
Running a test against `localhost` will resolve as the 127.0.0.1 inside the Tracetest container. To run tests against apps running on your local machine, add them to the same network and use service name mapping instead. Example: Instead of running an app on `localhost:8080`, add it to your Docker Compose file, connect it to the same network as your Tracetest service, and use `service-name:8080` in the URL field when creating an app.
```